lib/cuda/cudaCopyToRingbuffer.cpp

```diff
@@ -151,6 +151,16 @@ cudaEvent_t cudaCopyToRingbuffer::execute(cudaPipelineState& pipestate,
         auto tmp = std::make_shared<chordMetadata>();
         tmp->deepCopy(meta);
         meta = tmp;
+
+        // TODO: Don't set the metadata every time (to, hopefully, the same values). Set them once
+        // in the beginning only.
+        assert(output_cursor % meta->sample_bytes() == 0);
+        meta->set_fpga_seq_num(meta->get_fpga_seq_num()
+                               - meta->get_time_downsampling_fpga().at(0)
+                                     * (output_cursor / meta->sample_bytes()));
+        // This should hold most of the time. It will be wrong when we start Kotean with a non-zero
+        // `fpga_seq_num`.
```

Comment:
we do. This will fail, pretty much every single time I start kotekan on the test node since the synchronization that dpdk does will mean that the very first `fpga_seq_num` passed to kotekan will not be zero (in fact it's rather large for me).